src/components/Navbar/Navbar.tsx

```diff
@@ -119,8 +131,8 @@ export const Navbar = () => {
                   ))}
                 </Stack>
```

Comment:
The middle text  in the app bar currently lays over the items when you minimize to around 700px and although we have not decided on what's going there we should start laying out clickable buttons like so:
```suggestion
  <List sx={{display: 'flex', flexDirection: 'row', justifyContent: 'flex-end', width:'100%', gap: '4px'}}>
    {linkItems.map((text) => (
        <MenuItem sx={{p: 1}} key={text} disableGutters>
          {text}
        </MenuItem>
    ))}
  </List>
```

There's other changes we can make to make the middle text fit which I will leave a comment on.

Reply:
Good catch made the change